docs/bridging/bridging-fa-how.md

```diff
@@ -0,0 +1,97 @@
+- **ERC-20 proxy contract**: Stores tickets and mints ERC-20 tokens that are equivalent to the FA tokens in layer 1
+
+Examples of these contracts and tools to deploy them are available in the repository https://github.com/baking-bad/etherlink-bridge.
+
+## Depositing tokens from layer 1 to Etherlink
+
+The process of bridging FA-compatible tokens from layer 1 to Etherlink (also known as depositing tokens) follows these general steps:
+
+1. A Tezos user gives the token bridge helper contract access to their tokens.
+
+   - For FA1.2 tokens, the user gives the helper contract an allowance of tokens.
+   - For FA2 tokens, the user makes the helper contract an operator of their tokens.
+
+   For information about token access control, see [Token standards](https://docs.tezos.com/architecture/tokens#token-standards) on docs.tezos.com.
+
+1. The user calls the helper contract's `deposit` entrypoint.
+The request includes the amount of tokens to bridge, the address of the Etherlink Smart Rollup, and the user's Etherlink wallet address, but not the tokens themselves.
+
+1. The token bridge helper contract stores the address of the Etherlink Smart Rollup and the user's Etherlink address temporarily.
+
+1. The helper contract (as an operator of the user's tokens or with an allowance of the user's tokens) calls the token contract to transfer the tokens from the user's account to its account.
+
+1. The helper contract calls the ticketer contract's `deposit` entrypoint and includes the tokens.
+
+1. The ticketer contract stores the tokens and creates a [ticket](https://docs.tezos.com/smart-contracts/data-types/complex-data-types#tickets) that represents the receipt of the tokens.
+
+1. The ticketer contract sends the ticket back to the helper contract.
+
+1. The helper contract forwards the ticket to the Smart Rollup inbox and clears its storage for the next transfer.
+
+1. The Etherlink Smart Rollup receives the ticket and sends it to the ERC-20 proxy contract.
+
+1. The ERC-20 proxy contract stores the ticket, mints the equivalent tokens, and sends them to the user's Etherlink account.
+
+This diagram is an overview of the process of bridging tokens from layer 1 to Etherlink:
+
+![Overview of the FA token bridging deposit process](/img/bridging-deposit-fa.png)
+<!-- https://lucid.app/lucidchart/50249082-2195-40fa-8fa0-bd030ef6b12e/edit -->
+
+## Withdrawing tokens from Etherlink to layer 1
+
+The process of bridging FA-compatible tokens from Etherlink to layer 1 (also known as withdrawing tokens) follows these general steps:
+
+1. The user calls the FA withdrawal precompiled contract on Etherlink and includes this information:
+
+   - The address of the ERC-20 proxy contract that stores the ticket that represents the tokens
+   - The amount of tokens to bridge
+   - The user's layer 1 address or the address of a contract to send the tokens to
```

Comment:
The amount of tokens to bridge is right before the ticketer contract in the ABI if you care about the order

Reply:
Updated.